src/Transports/MassTransit.EventHubIntegration/Processors/EventHubDefinition.cs

```diff
@@ -0,0 +1,77 @@
+namespace MassTransit.EventHubIntegration.Processors
+{
+    using System;
+    using System.Collections.Generic;
+    using Configuration;
+    using Configurators;
+    using GreenPipes;
+    using Pipeline.Observables;
+    using Registration;
+
+
+    public class EventHubDefinition :
```

Comment:
We need a term other than definition, since that conflicts with the ConsumerDefinition, etc.